README.md

```diff
@@ -5,7 +5,10 @@ Tool for automatic synthesis of VPC network configurations, namely Network ACLs
 Multi-vpc input is supported. Required connections cannot cross vpc boundaries.
 
 ## Usage
-Use the `target` flag to specify the type of network resources to generate: whether Network ACLs or Security Groups.
+Use the `vpcgen` CLI tool with one of the following commands to specify the type of network resources to generate.
+* `vpcgen sg` - generate Security Groups.
+* `vpcgen acl` - generate an nACL for each subnet separately.
+* `vpcgen acl --single` - generate a single nACL for all subnets in the same VPC.
 
 ### nACLs Generation
 There is an option to generate an nACL for each subnet separately, or to generate a single nACL for all subnets in the same VPC.
```

Comment:
```suggestion
Specifying the `--single` flag results in generating a single nACL for all subnets in the same VPC. Otherwise, an nACL is generated for each subnet separately,
```